File: ServerSpace/src/main/java/Business/Servicio.java
package Business;

import EntityLayer.ServicioEntity;
import java.util.ArrayList;
import DataLayer.ServicioDB;
import Framework.Inj;

public class Servicio {

    public ArrayList<ServicioEntity> ObtenerMain() {
        try {
            ServicioDB BD = new ServicioDB();
            return BD.ObtenerMain();
        } catch (Exception ex) {
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }

    public ServicioEntity Registrar(ServicioEntity Item) {
        try {
            Inj.IniciarTranssaccion(true);
            ServicioDB BD = new ServicioDB();
            BD.Save(Item);
            Inj.FinalizarTranssaccion();
            return Item;
        } catch (Exception ex) {
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }

  
    public ArrayList<ServicioEntity> ObtenerItem(int Id) {
        try {
            ServicioDB BD = new ServicioDB();
            return BD.ObtenerItem(Id);
        } catch (Exception ex) {
            throw new UnsupportedOperationException("MENSAJE :" + ex);
        }
    }

}

File: SharedApi/Space/Space.BusinessLayer/Servicio.cs
using Framework;
using Space.DataLayer;
using Space.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Space.BusinessLayer
{
    public class Servicio
    {
        public List<ServicioEntity> ObtenerMain()
        {
            ServicioDB DB = new ServicioDB();
            return DB.ObtenerMain();
        }
        public Int32 Registrar(ServicioEntity Ent)
        {
            ServicioDB DB = new ServicioDB();
            DB.Registrar(Ent);
            return Ent.ServicioId;
        }
      
        public List<ServicioEntity> ObtenerItem(Int32 ServicioId)
        {
            ServicioDB DB = new ServicioDB();
            return DB.ObtenerItem(ServicioId);
        }

    }
}
